common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -363,6 +363,19 @@ public static int javaVersion() {
         return PlatformDependent0.javaVersion();
     }
 
+    /**
+     * @param thread the thread to be checked.
+     * @return
+     * {@code 0}: MUST be a virtual thread.
+     * <br>
+     * {@code 1}: MUST NOT be a virtual thread.
+     * <br>
+     * {@code -1}: Not able to check the thread type.
+     */
+    public static int checkVirtualThread(Thread thread) {
```

Comment:
we could also use an enum to make things easier to check

Reply:
Sounds good, will do it.